quesma/quesma/functionality/bulk/bulk.go

```diff
@@ -88,6 +99,11 @@ func Write(ctx context.Context, defaultIndex *string, bulk types.NDJSON, lm *cli
 			for _, document := range documents {
 				stats.GlobalStatistics.Process(cfg, indexName, document, clickhouse.NestedSeparator)
 			}
+			// if the index is mapped to specified database table in the configuration, use that table
+			sourceToDestinations := makeSourceToDestMappings(cfg.IndexSourceToInternalMappings)
+			if destIndex, ok := sourceToDestinations[indexName]; ok {
+				return lm.ProcessInsertQuery(ctx, destIndex, documents)
+			}
```

Comment:
This is probably out-of-scope of this PR, but I wonder if we could unify "ingest transformers" in the future. For example now there's a `RewriteArrayOfObject` "ingest transformer" already here:

https://github.com/QuesmaOrg/quesma/blob/ebeaa89c994b62c9f0d7a402f7a8fe441f63ac3b/quesma/clickhouse/clickhouse.go#L499-L512

Reply:
Probably we could do that, however this requires some changes in transformers API and as you mentioned this is out of scope of this particular PR